sim/utils.py

```diff
@@ -2,7 +2,7 @@
 import pandas as pd
 from io import StringIO
 
-def run(cmd, *args, **kwargs):
+def run(cmd, *args, capture_output=False, **kwargs):
```

Comment:
This `capture_output` keyword argument is unnecesary here, since all the `**kwargs` are passed into subprocess run

Reply:
Its funny, I wanted to have this in because i use `capture_output` later in the function. Without explicitly setting it an error is thrown because kwargs['capture_output'] doesn't exist.